README.md

```diff
@@ -54,13 +71,77 @@ The `azdev` tool is designed to aid new and experienced developers in contributi
     source env/bin/activate
     ```
 
-5. Install `azdev` by running:
-  `pip install azdev`
+5. Prepare and install `azdev`
+
+   If you're on Linux, install the dependency packages first by running:
+
+   For apt packages:
+   ```Bash
+   sudo apt install gcc python3-dev
```

Comment:
Does azdev require gcc?

Reply:
No. It's required by some dependencies of azure cli when installing them during `azdev setup`.